recipes/newrelic/apm/php/debian-fpm.yml

```diff
@@ -352,7 +352,7 @@ install:
             echo -e "{{.WHITE}}Running PHP-CLI with user ${SUDO_USER}{{.GRAY}}"
             for i in {1..10}; do
               sleep 1
-              sudo -u $SUDO_USER php —-ini
+              sudo -u $SUDO_USER php --ini
```

Comment:
Wow, what an obscure find.  Great job.

Reply:
It wasn't an issue with the first instance. That's part of what led me to finding this. I kept getting an error about a file not being found which was I could not figure out. I searched for `--ini` to find the line and eventually noticed that only one of the two occurrences was being highlighted.